### PR TITLE
TK3.3: add service args to hooks.event-hooks

### PR DESCRIPTION
**Pull request by frenchy64**
Related https://github.com/threatgrid/iroh/issues/3986

<a name="qa">[§](#qa)</a> QA
============================

No QA is needed.

<a name="release-notes">[§](#release-notes)</a> Release Notes
=============================================================

```
intern: TK3.3: add service args to event hooks.event-hooks
```

<a name="squashed-commits">[§](#squashed-commits)</a> Squashed Commits
======================================================================

### CODE REVIEW

File: src/ctia/flows/hooks.clj [hunk 1]
```diff
@@ -24,7 +25,7 @@
 (defn reset-hooks! []
   (reset! hooks
           (-> empty-hooks
-              event-hooks/register-hooks)))
+              (event-hooks/register-hooks p/get-in-global-properties))))
```
Comment (frenchy64) [inline src/ctia/flows/hooks.clj:28]: Hooks becomes its own TK service, so this is the "edge".

Comment (frenchy64) [inline src/ctia/flows/hooks.clj:28]: This is a convenient place to stop adding args and just reference the global properties service. Hooks becomes its own TK service which uses something like `reset-hooks!` in its `start` method.

File: test/ctia/flows/hooks/redismq_event_hook_test.clj [hunk 2]
```diff
@@ -25,12 +25,13 @@
                                     es-helpers/fixture-properties:es-store
                                     fixture-properties:redismq-hook
                                     test-helpers/fixture-properties:events-enabled
-                                    test-helpers/fixture-ctia
-                                    test-helpers/fixture-allow-all-auth]))
+                                    test-helpers/fixture-allow-all-auth
+                                    test-helpers/fixture-ctia]))
```
Comment (frenchy64) [inline test/ctia/flows/hooks/redismq_event_hook_test.clj:29]: Prep: `fixture-ctia` will be in charge of starting an `app`, so it must be last in order to pick up other fixtures in the ConfigService.

**Review by gbuisson — APPROVED** [review]
LGTM!

**Review by ereteog — APPROVED** [review]
lgtm

**Pull request merged**